torch_frame/utils/concat.py

```diff
@@ -11,11 +12,27 @@
 from torch_frame.data.tensor_frame import TensorFrame
 from torch_frame.typing import TensorData
 
+T = TypeVar(
+    "T",
```

Comment:
What's the meaning of T here?

Reply:
I think `T` is a common name for a type variable although I'm open to renaming it :)

Reply:
Oh, sorry I mean the `"T"` in `TypeVar`. Didn't know that we should have `"T"` in `TypeVar` to match with the `T`